src/components/RecentAuctions/RecentAuctions.css

```diff
@@ -0,0 +1,36 @@
+@media all and (max-width: 480px) {
+    .cardWrapper {
+        padding: 15px;
+        width: 100%;
+        display: inline-block;
```

Comment:
padding: 15px; and `display: inline-block;` are redundant

Reply:
cant be helped